build.cake

```diff
@@ -234,50 +230,8 @@ Task("RefDocs")
     });
 
 Task("Package")
-    .IsDependentOn("PackageReleaseZipFile")
     .IsDependentOn("PackageNugetPackages");
 
-Task("PackageReleaseZipFile")
```

Comment:
Remove this task.

Reply:
By removing this task, I believe package no longer implicitly calls BuildArtifacts and BuildApiDocs: are we okay with this? I think it's fine since it saves an unneeded step when creating patch releases

Reply:
I thought about that... but the `Package` test never did depend on those two tasks. Only `PackageReleaseZipFile` depended on them and that task no longer exists.

Reply:
In theory we *could* have removed this `Task` since it only depends on one other `Task`, but it's easier to type

    .\build.ps1 -target package

than

    .\build.ps1 -target packagenugetpackages